packages/trace-viewer/src/ui/uiModeView.tsx

```diff
@@ -607,11 +620,13 @@ const refreshRootSuite = (eraseResults: boolean): Promise<void> => {
         ++progress.failed;
       else
         ++progress.passed;
-      throttleUpdateRootSuite(config, rootSuite, progress);
+      throttleUpdateRootSuite(config, rootSuite, loadErrors, progress);
     },
 
     onError: (error: TestError) => {
       xtermDataSource.write((error.stack || error.value || '') + '\n');
+      loadErrors.push(error);
```

Comment:
This will keep pushing new (probably duplicate) errors after the initial load.

Reply:
Let me see if I can mute it.

Reply:
Not getting a good idea about it. But willing to wait for a report.